datafusion/core/tests/parquet/mod.rs

```diff
@@ -66,7 +66,9 @@ enum Scenario {
     Int32Range,
     Float64,
     Decimal,
+    DecimalBloomFilter,
```

Comment:
There are only two scenarios here but three encodings (Int32, Int64 and Bytes) handled by the code. Is there some way to over all three encodings?

Reply:
add test